src/api/search.api.js

```diff
@@ -272,6 +281,141 @@ async function searchLayerFeatures(outputProjection, queryString, layer, lang, c
     }
 }
 
+/**
+ * Searches for the query string in the feature inside the provided search fields
+ *
+ * @param feature
+ * @param {String} queryString
+ * @param {String} searchFields
+ * @returns {Boolean}
+ */
+function isQueryInFeature(feature, queryString, searchFields) {
+    const queryStringClean = queryString
+        .trim()
+        .toLowerCase()
+        .normalize('NFD')
+        .replace(/[\u0300-\u036f]/g, '')
```

Comment:
what are these two specific chars? (this needs a little comment explaining what you are doing 😉 )